src/python/gudhi/subsampling.pyx

```diff
@@ -27,6 +27,11 @@ cdef extern from "Subsampling_interface.h" namespace "Gudhi::subsampling":
     vector[vector[double]] subsampling_sparsify_points(vector[vector[double]] points, double min_squared_dist)
     vector[vector[double]] subsampling_sparsify_points_from_file(string off_file, double min_squared_dist)
 
+cdef extern from "Subsampling_interface.h":
+    cdef int _GUDHI_SUBSAMPLING_USE_CGAL
+
+GUDHI_SUBSAMPLING_USE_CGAL = _GUDHI_SUBSAMPLING_USE_CGAL
```

Comment:
Is it useful to have both GUDHI_SUBSAMPLING_USE_CGAL and _GUDHI_SUBSAMPLING_USE_CGAL? (I didn't think long about it)

Reply:
I could do it this way:
```python
cdef extern from "Subsampling_interface.h":
    cdef enum:
        _GUDHI_SUBSAMPLING_USE_CGAL
```
Then in cython `if not _GUDHI_SUBSAMPLING_USE_CGAL:` would work, but the information wouldn't be available at python level (required for unitary tests purpose)